pkg/webui/console/components/device-data-form/index.js

```diff
@@ -177,13 +177,13 @@ class DeviceDataForm extends Component {
           required
         />
         <Form.Field
-          title={sharedMessages.fwdNtwkKey}
+          title={sharedMessages.nwkSKey}
```

Comment:
It looks like we're losing information here

Reply:
`<DeviceDataForm />` is not related here. The only place we use it is in https://github.com/TheThingsNetwork/lorawan-stack/blob/feature/579-device-general-settings/pkg/webui/console/views/device-general-settings/network-server-form/index.js#L258

Is this correct?